testpilot/frontend/static-src/styles/_layout.scss

```diff
@@ -85,3 +85,13 @@
 .overflow-hidden {
   overflow: hidden;
 }
+
+.show-on-small {
+  @include respond-to('not-small') {
+    display: none;
+  }
+
+  @include respond-to('small') {
```

Comment:
No need for this media query. Just have it be block by default, and override when not-small.